cmd/deptool/scanner.go

```diff
@@ -102,6 +102,9 @@ func addTomlDependencies(dependencies *projectDependencies, tomlDeps map[string]
 			fmt.Printf("Conflicting entries in Cargo.toml file :\n%v\nvs.\n%v\n", existing, current)
 			exitErr()
 		}
+		if current.githubPath == "" {
+			continue
```

Comment:
could guard check earlier after line 92 for `crateGit.Git = ""` instead ?

Reply:
good point. updated.